detox/src/devices/Device.js

```diff
@@ -42,6 +43,16 @@ class Device {
     await this.client.sendUserNotification(params);
   }
 
+  async setOrientation(orientation) {
+    // orientation is 'landscape' (meaning left side portrait) or 'portrait' (non-reversed)
+    const call = invoke.call(
```

Comment:
Sure, I just wanted to state that calling 
```js
invoke.IOS.EarlGrey(
         'rotateDeviceToOrientation:errorOrNil:',
        invoke.IOS.NSInteger(orientationEnum)
);
``` 

would be the most readable and easiest way, I will try to make it work, the tests will tell me if I did something wrong then ;)